<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>1199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>720</width>
      <height>1280</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 228, 229);</string>
    </property>
    <widget class="QWidget" name="widget_6" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>671</width>
       <height>541</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 171, 172);
border: 4px solid black;
</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>431</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 36pt &quot;MS Shell Dlg 2&quot;;
border: 0px;</string>
      </property>
      <property name="text">
       <string>Medication Plan</string>
      </property>
     </widget>
     <widget class="QPushButton" name="GoBackMed">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>450</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: white;
border-radius:20px;
font: 26pt &quot;MS Shell Dlg 2&quot;;

</string>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="GoForwardMed">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>450</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: white;
border-radius:20px;
font: 26pt &quot;MS Shell Dlg 2&quot;;

</string>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="Name">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>351</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;

color: rgb(0, 0, 0);
border: 0px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Effect">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>250</y>
        <width>351</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;

color: rgb(0, 0, 0);
border: 0px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Dosage">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>310</y>
        <width>351</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;

color: rgb(0, 0, 0);
border: 0px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Status">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>571</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
border: 0px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>30</y>
       <width>731</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 171, 172);
</string>
     </property>
     <widget class="QPushButton" name="GoBack">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: white;
border-radius:20px;
font: 26pt &quot;MS Shell Dlg 2&quot;;

</string>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>551</width>
        <height>61</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 28pt &quot;MS Shell Dlg 2&quot;;
text-align: center;
border:0px;</string>
      </property>
      <property name="text">
       <string>Medical Health System</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="UpdateValues">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>730</y>
       <width>351</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 171, 172);
border: 4px solid black;
border-radius:20px;
font: 30pt &quot;Monospace&quot;;</string>
     </property>
     <property name="text">
      <string>Update Values</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Config.qrc"/>
 </resources>
 <connections/>
</ui>
